_inc/client/traffic/search.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { translate as __ } from 'i18n-calypso';
+import includes from 'lodash/includes';
+
+/**
+ * Internal dependencies
+ */
+import { ModuleSettingsForm as moduleSettingsForm } from 'components/module-settings/module-settings-form';
+import { ModuleToggle } from 'components/module-toggle';
+import SettingsCard from 'components/settings-card';
+import SettingsGroup from 'components/settings-group';
+import { getSiteAdminUrl } from 'state/initial-state';
+import { getPlanClass } from 'lib/plans/constants';
+import { getSitePlan } from 'state/site';
+import { isFetchingSiteData } from 'state/site';
+
+const Search = moduleSettingsForm(
+	React.createClass( {
+		render() {
+            const search = this.props.getModule( 'search' );
+            let planClass = null;
+
+            if ( this.props.sitePlan ) {
+                planClass = getPlanClass( this.props.sitePlan.product_slug );
+            }
+
+            if ( includes( [ 'is-personal-plan', 'is-premium-plan', 'is-business-plan' ], planClass ) ) {
+                return (
+                    <SettingsCard
+                        { ...this.props }
+                        module="search"
+                        hideButton
+                    >
+                        <SettingsGroup module={ { module: 'search' } } hasChild support={ search.learn_more_button }>
+                            <ModuleToggle
+                                slug="search"
+                                compact
+                                activated={ this.props.getOptionValue( 'search' ) }
+                                toggling={ this.props.isSavingAnyOption( 'search' ) }
+                                toggleModule={ this.props.toggleModuleNow }>
+                                { __( 'Enhanced site-wide search, powered by ElasticSearch (Beta)' ) }
```

Comment:
"ElasticSearch" should be "Elasticsearch". Let's try and get their branding right :)

Reply:
Fixed - thanks (plus I fixed it in th

Reply:
Thank you!